monkey/monkey_island/cc/services/initialize.py

```diff
@@ -153,7 +152,16 @@ def _register_services(container: DIContainer):
     container.register_instance(RepositoryService, container.resolve(RepositoryService))
 
 
-def _patch_credentials_parser(container: DIContainer):
+def _dirty_hacks(container: DIContainer):
+    # A dirty hacks function that patches some of the things that
+    # are needed at the current point
+
+    # Patches attack technique T1003 which is a static class
+    # but it needs stolen credentials from the database
+    T1003.get_report_data = container.resolve(T1003GetReportData)
```

Comment:
Do we have a plan to fix this in the future?

Reply:
Yep. This is just a temporary hack for now. @mssalvatore knows more about the plan